mypy/util.py

```diff
@@ -290,6 +290,22 @@ def unmangle(name: str) -> str:
     return name.rstrip("'")
 
 
+def get_unique_redefinition_name(name: str, existing: Container[str]) -> str:
```

Comment:
I wonder if the mangled names can be shown in error messages (say, when showing error context). If you believe that it might be the case, can you create a follow-up issue about it?

Reply:
Out of top of my head I found only one case (multiple inheritance) where we use names from symbol table instead of `.name()` or `.fullname()`, but there may be other. I will open an issue.